### PR TITLE
Add sysctls option to docker_swarm_service

### PR DESCRIPTION
**Pull request by NairolfL**
##### SUMMARY
Add sysctls options to docker_swarm_service, as discussed in #190 

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME
docker_swarm_service

### CODE REVIEW

**Comment by NairolfL** [comment]
This is my first time contributing to this project. Don't hesitate to tell me if I'm missing something / doing something wrong.

**Comment by felixfontein** [comment]
You also need to adjust the integration tests. You probably have to add `sysctls: null` to https://github.com/ansible-collections/community.docker/blob/main/tests/integration/targets/docker_swarm_service/vars/main.yml.

File: plugins/modules/docker_swarm_service.py [hunk 1]
```diff
@@ -83,6 +83,11 @@
       - Dictionary of key value pairs.
       - Corresponds to the C(--container-label) option of C(docker service create).
     type: dict
+  sysctls:
+    description:
+      - Dictionary of key, value pairs.
+    version_added: 3.9.0
```
Comment (felixfontein) [inline plugins/modules/docker_swarm_service.py:89]:
```suggestion
    version_added: 3.10.0
```

**Comment by NairolfL** [comment]
Thank you for the feedback, I made the requested changes

**Comment by felixfontein** [comment]
Thanks, looks good to me! If nobody objects, I'll merge this in ~a week.

**Comment by felixfontein** [comment]
@NairolfL thanks for your contribution!

**Pull request merged**